digitalforms-api/src/main/java/ca/bc/gov/open/pssg/rsbc/digitalforms/controller/UtilityController.java

```diff
@@ -34,19 +35,29 @@
 	private final Logger logger = LogManager.getLogger(UtilityController.class);
 	
 	@Autowired
-	private HealthApi healthApi;
+	private HealthApi vipsHealthApi;
 	
+	@Autowired
+	private ca.bc.gov.open.pssg.rsbc.digitalforms.ordsclient.api.HealthApi digitalFormsHealthApi;
+	
+	@SuppressWarnings("unchecked")
 	@ApiOperation(value = "Digital Forms Ping Service", response = PingResponse.class)
 	@ApiResponses(value = {@ApiResponse(code = 200, message = "Success", response = PingResponse.class) })
 	@GetMapping(path ="/api/utility/ping",
 	produces = "application/json"
 	)
-	public ResponseEntity<PingResponse> getPing() throws ApiException {
-		
+	public ResponseEntity<PingResponse> getPing() throws ApiException, ca.bc.gov.open.pssg.rsbc.digitalforms.ordsclient.api.handler.ApiException {
```

Comment:
A comment... Can we remove the first ApiException type from this line and just use the later?

Reply:
Both ApiExceptions are checked exceptions for each Vips ords and digital forms ords. Hence I dont think it can be removed

Reply:
Fine, Thanks, Just checking.

Reply:
As a final note.. We'll have to start adding proper exception handling when we get more into wiring up the VIPS/Digiatl Forms API so we'll need to create a digital forms exception type in the project. Something for later.

Reply:
sounds good